doc/architecture/decisions/0002-formatting-rules.md

```diff
@@ -0,0 +1,120 @@
+# 2. Formatting Rules
+
+Date: 2025-11-10
+
+## Status
+
+Accepted
+
+## Context
+
+The Swedish Pathogens Portal codebase needs consistent formatting rules to ensure code readability, maintainability, and ease of code reviews. Currently, some formatting rules are documented (Python via Ruff configuration), while others are implicit or undocumented (HTML templates, file naming, URLs, docstrings).
+
+Formalizing these rules will:
+- Improve code consistency across the project
+- Make code reviews more efficient
+- Reduce formatting debates during development
+- Provide clear guidelines for new team members
+- Ensure better developer experience
+
+## Decision
+
+We will adopt the following formatting rules for the Swedish Pathogens Portal project:
+
+### HTML Line Length
+- **Rule**: Maximum 80 characters per line in HTML template files, primarily referring to content/text displayed on webpages (e.g., text within `<p>`, `<h1>`, `<div>`, and other content tags)
+- **Rationale**: Improves readability in code reviews, easier to view in terminal/IDE side-by-side, aligns with common HTML style guides. Prevents overly long lines that are hard to scan, especially for content that will be displayed to users.
+- **Note**: While HTML templates with TailwindCSS utility classes can create long lines in code, the 80-character limit primarily applies to visible content/text. Code formatting (attributes, class lists) may exceed this limit when necessary for readability.
+- **Enforcement**: Manual code review. Automated tooling (e.g., Prettier, HTML linters) may be considered in the future to enforce this rule.
+
+### File Naming Convention
+- **Rule**: Use lowercase with underscores for all file names (e.g., `my_file_name.html`, `topic_detail.py`)
+- **Rationale**: 
+  - Consistent with Python conventions (PEP 8)
+  - Consistent with Django best practices
+  - Avoids case-sensitivity issues across operating systems
+- **Examples**:
+  - ✅ `models.py`, `topic_detail.html`, `article_detail.html`
+  - ❌ `Models.py`, `TopicDetail.html`, `article-detail.html`
+- **Current State**: Already followed throughout the codebase
+
+### URL Naming Convention
+- **Rule**: Use dashes (hyphens) in URL paths (e.g., `/my-website-url/`, `/topic-detail/`)
```

Comment:
Maybe we should also mention "lower case" like how it was mentioned above (in the file name convention).